### PR TITLE
Some Poller page cleanups

### PR DESCRIPTION
**Pull request by murrant**
Some queries loaded all devices into memory and the counted them, we should have sql count them if we only need a count.
Models should be singular
Use named routes for url generation
Try to keep presentation and data collection separated in blade and controller a little more
Don't use authorize if it isn't wired up.

@SourceDoctor I also noticed all the ajax calls are to legacy code ajax_table and ajax_form.

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by SourceDoctor** [comment]
yeah i know, Still much to do for rewriting.
Maybe create a new similar to [this](https://github.com/librenms/librenms/pull/8703) for ajax calls and going through the system like other tasks in the [Project](https://github.com/librenms/librenms/projects/4)

File: app/Http/Controllers/PollerController.php [hunk 1]
```diff
@@ -5,26 +5,18 @@
...
-        'id' => 0,
-        'group_name' => 'General',
-        'descr' => ''
-    ];
-    public $defaultPollerMarker = '(default Poller)';
 
     public function __construct()
     {
-        $this->authorizeResource(PollerGroups::class, 'poller_groups'); // FIXME is this correct? not a resource anymore
         $this->rrdstep = \LibreNMS\Config::get('rrd.step');
```
Comment (SourceDoctor) [inline app/Http/Controllers/PollerController.php]: could also be changed to `Config::get('rrd.step');` :)

Comment (murrant) [reply app/Http/Controllers/PollerController.php]: Oops, It was, but then I changed things around and it got reverted :)

**Comment by murrant** [comment]
@SourceDoctor  Yeah, I know the feeling it is like a rabbit hole.

**Pull request merged**